src/Avalonia.Controls/RenderOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Avalonia.Controls
+{
+    using Avalonia.Visuals.Media.Imaging;
```

Comment:
I think you should make the `using` before the namespace to hold the same style.